code/controllers/subsystems/machinery.dm

```diff
@@ -69,13 +69,13 @@
 
 		var/start_tick = world.time
 
-		if (M.isprocessing)
+		if (M.machinery_processing)
 			processes_this_tick++
 			switch (M.process())
```

Comment:
![2017-05-27_15-41-18](https://cloud.githubusercontent.com/assets/7097800/26524258/f33e3a16-42f2-11e7-8fce-eb06fe14d864.png)
Main reason against is having to update all the machinery `process()` procs (i.e. laziness)

Probably a good idea though.